src/components/NewPipelineBox/index.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from "react";
 import { Modal, Row, Col, Card, Tag, Button } from 'antd';
 
-import { PIPELINE_TEMPLATES } from '~/config/index';
-import { pipeline } from '~/common/service';
+import { PIPELINE_TEMPLATES } from 'src/config/index';
```

Comment:
Why do we deprecate the usage of ~?

Reply:
The `~` is used by Parcel to resolve tilde paths(the nearest directory with package.json or the project root).

doc: https://v2.parceljs.org/features/module-resolution/#aliases